selfdrive/car/honda/interface.py

```diff
@@ -181,9 +181,19 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), has_relay=False,
       ret.wheelbase = CivicParams.WHEELBASE
       ret.centerToFront = CivicParams.CENTER_TO_FRONT
       ret.steerRatio = 15.38  # 10.93 is end-to-end spec
-      ret.lateralParams.torqueBP, ret.lateralParams.torqueV = [[0, 4096], [0, 4096]] # TODO: determine if there is a dead zone at the top end
+      if eps_modified:
+        # stock request input values:     0x0000, 0x0067, 0x0107, 0x01CB, 0x0294, 0x035E, 0x0457, 0x060D, 0x06EE (STEER_CONFIG_INDEX: 1)
+        # stock request output values:    0x0000, 0x0380, 0x0800, 0x0C00, 0x0EB6, 0x10AE, 0x1200, 0x1200, 0x1200
+        # modified request output values: 0x0000, 0x0380, 0x0800, 0x0C00, 0x0EB6, 0x10AE, 0x1200, 0x1B00, 0x2D00
```

Comment:
does stock firmware clips request at `0x67F`? if yes, `0x2D00` is more than 2x torque at `0x67F`
```
>>> import numpy as np
>>> np.interp([0x67f], [0x060D, 0x06EE], [0x1B00, 0x2D00])array([9246.72])
>>> hex(9246>>1)
'0x120f'
```